usecases/schema/executor.go

```diff
@@ -237,12 +260,19 @@ func (e *executor) TriggerSchemaUpdateCallbacks() {
 	e.callbacksLock.RLock()
 	defer e.callbacksLock.RUnlock()
 
-	s := e.schemaReader.ReadOnlySchema()
+	wg := sync.WaitGroup{}
```

Comment:
please limit the wait group to for example `_NUMCPU * 2`

Reply:
one go routine is used now